fili-core/src/main/java/com/yahoo/bard/webservice/logging/JsonLogFormatter.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.logging;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A log formatter that prints log in JSON format.
+ */
+public class JsonLogFormatter implements LogFormatter {
+    private static final Logger LOG = LoggerFactory.getLogger(RequestLog.class);
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
This `ObjectMapper` should be configured with the [same modules](https://github.com/yahoo/fili/pull/81/files#diff-79f5168b37854c3172bc2994b14d3168R77) as the original mapper in `RequestLog` (which incidentally also needs to be removed).